### ISSUE TITLE
Ability to sort nodes by config order?

### ISSUE THREAD

**Issue by sparanoid** [valeriansaliou/vigil#101]
It seems all nodes are sorted alphabetically. Currently I can hack the order by prefixing numbers like:

```cfg
[[probe.service]]
id = "0-frontend"

...

[[probe.service]]
id = "1-backend"
```

But this also affects readability in notifications:

```
❌ Status changed to: DEAD.
- 0-frontend:homepage: 1 dead
Link: https://example.com/
```

That would be great if there is a way to sort them manually or just keep the order defined in the config.

**Comment by valeriansaliou** [comment]
That's right, prefixing node & service identifiers w/ numbers is the only way to go for now. Keeping this open, as this would be a better idea to have indexes there.
